apache2/msc_logging.c

```diff
@@ -1,2282 +1,2285 @@
-/*
```

Comment:
This is a huge diff :) Please make sure to not make any change in the tabulation or in the line terminator, otherwise it is very difficult to read the patch.